source/client-side-encryption/tests/README.rst

```diff
@@ -1415,7 +1423,7 @@ Four mock KMS server processes must be running:
 
 4. The mock `KMS KMIP server <https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/csfle/kms_kmip_server.py>`_.
 
-Create the following four ``ClientEncryption`` objects.
+Create the following ``ClientEncryption`` objects.
```

Comment:
Do the requirements for the "11. KMS TLS Options Tests" prose test need to change? I'm not sure if prose tests have historically referred to requirements beyond server version, but the PR description noted that this functionality depended on [MONGOCRYPT-605](https://jira.mongodb.org/browse/MONGOCRYPT-605).

Also, given that this is an extensive change to an existing prose test, you may want to consider striking the original test and appending this as a new test. The [prose test numbering rules](https://github.com/mongodb/specifications?tab=readme-ov-file#prose-test-numbering) don't really address this, so I won't push for anything.

Reply:
A new case within the prose test sounds like a good compromise.